### ISSUE TITLE
GitHub and Git tabs always open at launch, and no longer opens in separate pane

### ISSUE THREAD

**Issue by tz277** [atom/github#2203]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

GitHub and Git tabs not behaving as expected. 

### Steps to Reproduce

From what I remember:
1. Summon the GitHub Tab and Git Tab via Ctrl+Shift+9
2. Drag the GitHub Tab into it's own pane.

After this issue appears once; the issue appears everytime at launch. Furthermore, doing a full reinstall of Atom (eg following these steps: https://flight-manual.atom.io/faq/sections/how-do-i-uninstall-atom-on-macos/  and then installing again) does not fix the issue, and the abnormal behavior appears at launch. 

**Expected behavior:**

Normally (how it's always been since I started using the feature):

- Git/GitHub Tabs should not open at launch. 
- Ctrl+Shift+9 will toggle both GitHub/Git Tabs to appear/disappear together in a new pane. 
- Clicking between staged/unstaged changes on the Git Tab should open difference tabs on a pane that is not the one occupied by the GitHub/Git Tabs.

**Actual behavior:**

Now:
- Git/GitHub Tabs do open at launch. 
- Ctrl+Shift+9 only toggles the Git Tab and the Git Tab appears in the current pane.
- Clicking between staged/unstaged changes on the Git Tab opens difference tabs on the same pane occupied by Git Tab. 
  - This is inconvenient because it means I have to switch back to the Git Tab after each time I click to see changes. It's much nicer to keep the Git Tab in it's own pane and have changes appear in a pane that is not the one occupied by the Git Tab. 

**Reproduces how often:**

Every time. 

After this issue appears once; the issue appears everytime at launch. Furthermore, doing a full reinstall of Atom (eg following these steps: https://flight-manual.atom.io/faq/sections/how-do-i-uninstall-atom-on-macos/  and then installing again) does not fix the issue, and the abnormal behavior appears at launch. 

### Versions

```Atom    : 1.38.2
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3```

macOS Mojave version 10.14.5

![image](https://user-images.githubusercontent.com/39288574/61068108-34c2f980-a3d7-11e9-9cc5-6aa0ed735275.png)



### Additional Information

Thanks! :)

**Comment by vlmironoff** [comment]
Had the same issue. Try dragging the github tab to the bottom of the atom window and when it becomes a panel collapse it and it shouldn't pop open when not needed.

**Comment by chrisgo** [comment]
I would also like the git and github panes to not load up on the right when opening Atom (should be hidden until necessary).  

The only way to make this not work like this is to disable the entire package (not ideal).

Perhaps this should be a setting in the package itself to toggle [ ] Display git and github panes at startup and you can default it to checked and then allow people to disable it.  This is going to be better than having to disable the entire package completely ...

**Comment by smashwilson** [comment]
> Perhaps this should be a setting in the package itself

Nah, this is completely unintentional and shouldn't ever be happening. The only time we _intend_ the git and GitHub tabs to be open on launch are when:

* You launch Atom for the first time with the GitHub package available, but the Welcome pane is _not_ being shown.
* You had them open when you closed Atom in this session (where a session is governed by the set of root project folders).

You can see the original implementation way back in #875. At some point this broke and it became much more aggressive about opening on launch. This is a bug.

I don't have time to dig in to this soon but I'd love some help if anyone else wants to dig in :smile: The "initial open" behavior is managed by `lib/github-package.js` and `lib/controllers/root-controller.js`, I believe.

> Clicking between staged/unstaged changes on the Git Tab opens difference tabs on the same pane occupied by Git Tab.

Reading the original issue this feels like a different effort ☝️

**Comment by JeffreyDD** [comment]
I have this exact same issues and I'm seriously considering VS Code at this moment. For my, but I'd say any Users' use case, the Git tab is completely useless if it is not rendered in a separate pane, and shows changes in the same pane as the Git tab is rendered. Also did the complete reinstall, problem persists. Are we really the only ones dealing with this issue?

**Comment by ak4702** [comment]
I've just encountered the same issue and solved by myself.

My Atom has a GitHub button (which I don't use) at the bottom-right corner that opens normally, and the Git button that was used all the time and is screwed.

I opened the GitHub tab (that functions normally instead of opening up as another tab) and drag the Git tab back into the section. Problem solved.

**Comment by eposse** [comment]
Same issue here. Incidentally I also have the "Outline" tab always opened at startup.

@ak4702 can you clarify what you mean by "drag the Git tab back into the section"? What section?

**Comment by kaptron** [comment]
@eposse the solution above also worked for me. Use the Github button to open the right pane, and then drag the "unanchored" Git pane back into the right pane. Attached a screenshot. I really wonder if there's a config setting that is the equivalent to performing this action, but I couldn't find anything relevant. 

![Git_Atom](https://user-images.githubusercontent.com/318100/91646173-fc82de00-ea11-11ea-9da7-1d11e91a4080.jpg)

**Comment by ak4702** [comment]
> @eposse the solution above also worked for me. Use the Github button to open the right pane, and then drag the "unanchored" Git pane back into the right pane. Attached a screenshot. I really wonder if there's a config setting that is the equivalent to performing this action, but I couldn't find anything relevant.
> 
> ![Git_Atom](https://user-images.githubusercontent.com/318100/91646173-fc82de00-ea11-11ea-9da7-1d11e91a4080.jpg)

Precisely what I'm saying!

**Comment by Xliff** [comment]
This solution also works for me. How can this made to be the default behavior, because If I switch to a new file and open the git controls, it once again _replaces_ the file content, instead of putting them in their own pane.

**Comment by MaybeItsAdam** [comment]
You can drag the panels back into their original position in the side tab by dragging them to the right hand side of the screen and into the side panel that should open automatically on approach.
The arrows needed to open it automatically, however are hidden by some themes such as "One Dark" so my solution was to 

**change the UI theme to "Atom Dark" and drag the panels into place on the right hand side, the panels will remain there as you return to your original UI theme** 

This makes it an issue with the "One Dark" theme which removes the arrows and therefore the ability to return them intuitively to their original position

![image](https://user-images.githubusercontent.com/100949524/172223914-684dec9c-cc18-4d71-bad0-b9a23de2bc4f.png)

Hope this helps
